util/tagrepos.py

```diff
@@ -0,0 +1,46 @@
+from github import Github, GithubException
+import sys
+import csv
+
+if len(sys.argv) <> 3:
+    print "Please specify both username and github password."
+    print "Usage: tagrepos.py <username> <password>"
+    print "Takes CSV formatted stream as input following the format: "
+    print "github-reponame, sha1, tag"
+    print "Example:"
+    print "jbosstools/jbosstools-hibernate, a1f287bf825a891901b41739f6a4bbaafc70bcdd, jbosstools-4.2.1.Alpha23"
+    print
+    print "Useful when combined with buildinfo.json: "
+    print "cat buildinfo.json | python buildinfo_to_tags.py -n <tagname> | python tagrepos.py <taggeruser> <secret>" 
+    sys.exit(-1)
+
+repos = csv.reader(sys.stdin)
```

Comment:
you need to document usage to include the mention of the fact that you're simply piping the file into python instead of specifying it on the commandline. Incidentally, why do you feel a pipe is easier to use than "-f filename" ? Just curious.